app/assets/stylesheets/active_admin/components/_flash_messages.scss

```diff
@@ -1,23 +1,18 @@
 body.logged_in {
   .flash {
-    @include gradient(#f7f1d3, #f5edc5);
-    @include text-shadow(#fafafa);
-    border-bottom: 1px solid #eee098;
+    background-color: #f7f1d3;
     color: #cb9810;
     font-weight: bold;
     font-size: 1.1em;
-    line-height: 1.0em;
     padding: 13px 30px 11px;
     position: relative;
 
     &.flash_notice {
-      @include gradient(#dce9dd, #ccdfcd);
-      border-bottom: 1px solid #adcbaf;
+      background-color: #ccdfcd;
```

Comment:
Color literals like `#ccdfcd` should only be used in variable declarations; they should be referred to via variable everywhere else.